content/200-concepts/050-overview/500-under-the-hood.mdx

```diff
@@ -56,7 +56,7 @@ As an example, Prisma Client connects to the [query engine](../components/prisma
 
 ### Using custom engine binaries
 
-By default, all binaries are automatically downloaded into the `node_modules/@prisma/cli` folder when you install or update `@prisma/cli`. You might want to use a [custom binary](https://github.com/prisma/prisma-engines) file if:
+By default, all binaries are automatically downloaded into the `node_modules/@prisma` folder when you install or update `prisma`. You might want to use a [custom binary](https://github.com/prisma/prisma-engines) file if:
```

Comment:
This still has an `@` too much.

Reply:
I fixed this via this PR: https://github.com/prisma/docs/pull/1260

Note that the file paths here also still have an `@`, I'd assume but wasn't sure if they needed to be removed as well:

![image](https://user-images.githubusercontent.com/4058327/106869894-bfadc800-66d0-11eb-8d9e-e0b79a13a72c.png)

Maybe @ruheni you can validate and potentially fix the files paths if needed?

Reply:
The paths don't need to be fixed. The `@` is still part of the path to the packages in the `node_modules`
```bash
❯ npx prisma -v
Environment variables loaded from .env
prisma               : 2.16.0
@prisma/client       : 2.16.0
Current platform     : windows
Query Engine         : query-engine 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\query-engine-windows.exe)
Migration Engine     : migration-engine-cli 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine : introspection-core 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary        : prisma-fmt 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\prisma-fmt-windows.exe)Studio               : 0.346.0
```

Reply:
Yep, this is coming from the `@prisma/engines` package.